### PR TITLE
feat(metrics): add sampled (de)serialization duration metrics for RPC

### PR DESCRIPTION
**Pull request by Sunt-ing**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

***PLEASE DO NOT LEAVE THIS EMPTY !!!***
Add sampled (de)serialization duration metrics for RPC with the given frequency

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Refer to a related PR or issue link (optional)
https://github.com/singularity-data/risingwave/issues/3284

### CODE REVIEW

**Comment by Sunt-ing** [comment]
image

File: grafana/risingwave-dashboard.py [hunk 1]
```diff
@@ -385,6 +390,16 @@ def section_streaming_actors(outer_panels):
                     "rate(stream_executor_row_count[15s]) > 0", "{{actor_id}}->{{executor_id}}"
                 ),
             ]),
+            panels.timeseries_ns("Actor Sampled Deserilization Time", [
```
Comment (skyzh) [inline grafana/risingwave-dashboard.py:393]: typo

**Review by skyzh — COMMENTED** [review]
I believe this metrics is only used within scale service, and should not be put in dashboard.

**Comment by Sunt-ing** [comment]
No metric is used in all services. And since the panels of these two metrics locate in the "streaming actors" section, where most metrics that the scaling service needs locate, developers who do not care for scaling service can easily fold and unsee this section.

**Comment by skyzh** [comment]
The metrics you added do not convey any actual meaning. If developers observe "30ms" from the metrics, what can they know about the system?

**Comment by skyzh** [comment]
... and thus it's better not to add it to dashboard.

**Comment by Sunt-ing** [comment]
IMO, since sampling is used, the important information is not the accurate number but the existence and trend. If the graph of this metric exists, developers can know that this metric is not been disrupted. If this metric is growing, developers can know how the (de)serialization durations change. 
Also, IMO this Grafana panel is temporal and will be removed after the Orchestrator is fully built. And that, we may use other methods to observe it.

**Comment by skyzh** [comment]
> IMO, since sampling is used, the important information is not the accurate number but the existence and trend.

If we want to explore trend, we should use `rate` function.

> If this metric is growing, developers can know how the (de)serialization durations change.

It will change as long as there's an exchange.

> Also, IMO this Grafana panel is temporal and will be removed after the Orchestrator is fully built. And that, we may use other methods to observe it.

You can just leave it on your local branch, instead of merging into main.

**Comment by Sunt-ing** [comment]
With the rapid development, rebase can be miserable. Also, others who also develop the scaling service can benefit from it. I don't think this brings more overhead than its benefits.

**Comment by skyzh** [comment]
I'd only approve for the changes of metrics and not the dashboard. You can find another reviewer who agree on you.

**Comment by Sunt-ing** [comment]
Thanks for your review of the changes of metrics.

**Review by MingjiHan99 — APPROVED** [review]

**Pull request merged**